implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/v2/DynatraceExporterV2.java

```diff
@@ -198,18 +199,17 @@ private String createCounterLine(Meter meter, Measurement measurement) {
     }
 
     Stream<String> toTimerLine(Timer meter) {
-        if (!(meter instanceof DynatraceSummarySnapshotSupport)) {
-            // fall back to previous behaviour
+        if (!(meter instanceof TimeAwareDynatraceSummarySnapshotSupport)) {
             return toSummaryLine(meter, meter.takeSnapshot(), getBaseTimeUnit());
```

Comment:
Here we call `getBaseTimeUnit()`, but inside `meter.takeSnapshot()`, (DynatraceTimer:106) we do `baseTimeUnit()`. I'm a bit confusing how those differ? The former seems to come from our config but the other from MM? 🤔